tsconfig.json

```diff
@@ -12,6 +12,6 @@
     "esModuleInterop": true,
     "resolveJsonModule": true
   },
-  "exclude": ["dist", "node_modules"],
+  "exclude": ["dist"],
```

Comment:
Was this change intentional ?

Reply:
I'm not sure, it doesn't look like we should make this change. I reverted it in 1cd4f466280fdb8965009481aea44b791b15c13e.